teamcity/README.md

```diff
@@ -41,6 +42,15 @@ Add an item like the above to `instances` for each build configuration you want
 
 [Restart the Agent][6] to start collecting and sending Teamcity events to Datadog.
 
+#### Containerized
+For containerized environments, see the [Autodiscovery Integration Templates][1] for guidance on applying the parameters below.
+
+| Parameter            | Value                                                                                                                                         |
+|----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| `<INTEGRATION_NAME>` | `teamcity`                                                                                                                                    |
+| `<INIT_CONFIG>`      | blank or `{}`                                                                                                                                 |
+| `<INSTANCE_CONFIG>`  | `{"name": "<BUILD_NAME>", "server":"http://<USER>:<PASSWORD>@teamcity.<ACCOUNT_NAME>.com", "build_configuration":"<BUILD_CONFIGURATION_ID>"}` |
```

Comment:
seems this scheme is only valid if `basic_http_authentication` is true:
```
server: user:password@teamcity.mycompany.com # if you set basic_http_authentication to true
```

Reply:
Should it be changed to something different?

Reply:
I don't know how this is supposed to work but could it be `server: %%host%%` ?

We have `server: teamcity.mycompany.com` above

Reply:
`teamcity.mycompany.com` makes me think it's connecting to an external site. I will change to that.

Reply:
the thing is if it's connecting to an external website does it make sense to have AD documentation for it? 🤔

Reply:
Makes sense, updated